### DISCUSSION TITLE
How do I mount my server data directory using Samba

### DISCUSSION THREAD

**Discussion by Wezea** [nextcloud/all-in-one#5144] [Questions] [answered]
Hello,

I have tried for many hours to get this working but didn't succeed. 

I want to mount my data directory via Samba on my desktop machine. Therefore, I have created a dedicated samba user on the server (smbuser).
I have added it as a samba usr

`sudo smbpasswd -a smbuser`

The smb service is running with the following configuration:

```
[ncaio]
path = /home/user/nextcloud
writable = yes
browseable = yes
valid users = smbuser
force user = www-data
force group = www-data
# Whether I keep the following or leave it does not make a difference
create mask = 0770
directory mask = 0771
force create mode = 0660
force directory mode = 0770
```

On the desktop I have my credentials stored in `.smb` with the password for smbuser.

Then I try to mount this directory on my desktop like this:

`sudo mount -t cifs //<cloud-ip>/ncaio /home/desktopuser/ncaio -o rw,credentials=/home/desktop/.smb,vers=3.0`

I  run into a permission denied error. 

The samba log showing me that `www-data` trys to access the folder with uid=33 and gid=33 but gets rejected. However, the user and group `www-data` has r/w permission on that folder.  

I also added the user smbuser to the group `www-data`, but that also didn't work.

I am missing some Linux or docker fundamentals her, I guess. Do I somehow need to address the docker instance instead of trying to access the physical folder which is mounted by the container?

Thanks in advance

**Comment by szaimen** [comment]
Hi, maybe this helps?
https://github.com/nextcloud/all-in-one?tab=readme-ov-file#can-i-use-a-cifssmb-share-as-nextclouds-datadir

**Reply by Wezea** [reply]
Thanks for the support, but this describes how to use cifs/smb as a datadir. I want to share my existing data directory using smb with my desktop. I am aware of the synchronization issues, but I have a workflow which fits for me.

**Reply by szaimen** [reply, marked as answer]
Please open a thread then at https://help.nextcloud.com as this is not  specific AIO question.
